docs/luigi-client-api.md

```diff
@@ -384,6 +384,24 @@ Opens a view in a modal. You can specify the modal's title and size. If you don'
 LuigiClient.linkManager().openAsModal('projects/pr1/users', {title:'Users', size:'m'});
 ```
 
+#### openAsDrawer
+
+Opens a view in a drawer. You can specify if the drawer has a header, a backdrop is active in the background and the size of the drawer. By default the header is shown. The backdrop is not visible and has to be set to `true`. The size of the drawer is by default set to `m` which means 50% of the micro frontend size. You can also use `l`(75%) or `s`(25%). Optionally, use it in combination with any of the navigation functions.
```

Comment:
In `linkManager.js` (which I thought is the source of this document 🤔 ) the default size is specified as `s` instead of `m`. Please make sure they are aligned and the correct default value is used.

Reply:
Thx 👍  good point